src/components/Global/Footer.js

```diff
@@ -38,17 +38,17 @@ const navigation = [
 
 const Footer = () => {
   return (
-    <footer className="bg-transparent">
-      <div className="max-w-screen-xl px-4 pt-16 pb-6 mx-auto sm:px-6 lg:px-8 lg:pt-24">
-        <div className="grid grid-cols-1 gap-8 lg:grid-cols-3">
+    <footer className='bg-transparent'>
+      <div className='max-w-screen-xl px-4 pt-16 pb-6 mx-auto sm:px-6 lg:px-8 lg:pt-24'>
+        <div className='grid grid-cols-1 gap-8 lg:grid-cols-3'>
           <div>
-            <div className="flex justify-center text-teal-600 sm:justify-start">
-              <a href="#">
-                <span className="sr-only">WebX DAO</span>
+            <div className='flex justify-center text-teal-600 sm:justify-start'>
+              <a href='#' className=''>
```

Comment:
```suggestion
              <a href='#'>
```
No need of empty `className`